packages/server/test/course.integration.ts

```diff
@@ -563,4 +567,80 @@ describe('Course Integration', () => {
       expect(userCourse).toBeDefined();
     });
   });
+
+  describe('POST /register_courses', () => {
+    it('tests prof registering an array of courses', async () => {
+      const professor = await UserFactory.create();
+      await UserCourseFactory.create({
+        course: await CourseFactory.create(),
+        user: professor,
+        role: Role.PROFESSOR,
+      });
+      const course1: KhouryProfCourse = {
+        crns: [12345, 56765, 44444],
+        semester: '202210',
+        name: 'Underwater Basket-Weaving',
+      };
+      const course2: KhouryProfCourse = {
+        crns: [11111, 22222],
+        semester: '202230',
```

Comment:
These course names are great lmao, though we probably want to have the semester be the same between the both of them, because that's how things will work in practice. And you probably also wanna test that the LastRegistrationSemester has saved the current semester into it. Testing that the CRN mappings were created would probably be good as well.

Reply:
You got it boss